src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+    private String eventDate;
+    public Event (String description, String eventDate) {
+        super(description);
+        this.eventDate = eventDate;
+        System.out.println("  " + this.toString());
+    }
+
+    @Override
+    public String toString() {
+        String str = "[E]" + super.toString() +
+                "(at:" + this.eventDate + ")";
```

Comment:
it is good practice to break a long statement into several lines to increase readability.